.idea/misc.xml

```diff
@@ -22,7 +22,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_22" default="true" project-jdk-name="temurin-22" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_22" default="true" project-jdk-name="temurin-23" project-jdk-type="JavaSDK" />
```

Comment:
languageLevel="JDK_22 -> Shouldn't level also be 23?

Reply:
Nope, the default level when choosing 23 jdk becomes `JDK_X` which means experimental features.
Let me check with latest Intellij version to verify 100%.

Reply:
You were right. Updated to Intellij `2024.2.3` which has support for jdk 23 and it works as it should, with `default=true` and `languageLevel="JDK_23"`. thx! (updated the PR)